web/controllers/calendar_controller.ex

```diff
@@ -0,0 +1,25 @@
+defmodule CoursePlanner.CalendarController do
+  use CoursePlanner.Web, :controller
+
+  alias CoursePlanner.CalenderHelper
+  alias Ecto.Changeset
+
+  def index(%{assigns: %{current_user: current_user}} = conn, params) do
+    changeset = CalenderHelper.validate(params)
+
+    case changeset.valid? do
```

Comment:
You can have a cleaner structure if you pattern match `valid?` deconstructing `changeset`:
```
case CalenderHelper.validate(params) do
  %{valid?: true} = changeset -> do stuff
  %{errors: errors} -> do stuff
end
```